src/classes/StatsD.class.php

```diff
@@ -192,4 +191,67 @@ public function setHost($host)
         $this->host = $host;
     }
 
+    /**
+     * @param string $operation
+     * @param string|array $value
+     * @return array
+     */
+    protected function generateStatData($operation, $value)
+    {
+        $data = array();
+        foreach($this->getStatsPaths() as $path)
+        {
+            $data[$path . ".$operation"]=$value;
+        }
+        return $data;
+    }
+
+    /**
+     * @return string
+     */
+    public function getPivotValue()
+    {
+        return $this->pivotValue;
+    }
+
+    /**
+     * @param string $pivotValue
+     */
+    public function setPivotValue($pivotValue)
```

Comment:
Can't see where you could, but is it even possible to have a pivotValue of "."? Would that cause stats issues and do we need any checks on what this value can be?